python/lsst/ts/imsim/closedLoopTask.py

```diff
@@ -147,7 +147,7 @@ def _setSkySimBasedOnOpdFieldPos(self, instName, obsMetadata, parAngle, starMag)
         starId = 0
         raInDegArr = np.array(opdMetr.fieldX)
         decInDegArr = np.array(opdMetr.fieldY)
-        rotation = rotMatrix(obsMetadata.rotatorAngle - parAngle)
```

Comment:
Let's add a comment explaining this 180. (You can just copy and paste what I originally sent in slack).

Reply:
Done!